src/itensornetworkfunction.jl

```diff
@@ -80,6 +80,10 @@ function calculate_fxyz(
   alg=default_contraction_alg(),
   kwargs...,
 )
+  ## XXX: HACK UNTIL network fix
```

Comment:
Can we take this hack out ? It only fixes the case of a product state and you can just do `alg = " exact"` as a kwarg when calling the function to avoid the bug?